src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/quiet_but_failing.phpt

```diff
@@ -34,6 +34,3 @@ require __DIR__.'/fake_vendor/acme/outdated-lib/outdated_file.php';
 ?>
 --EXPECTF--
 Remaining indirect deprecation notices (1)
```

Comment:
This test was clearly designed to show the below message even when `verbose=0` so I'm guessing my "fix" is wrong but at least (hopefully) better demonstrates the problem: when any of the levels has a failure, all level output is verbose again - even levels that do not cause a failure.